tests/test_debugger.py

```diff
@@ -0,0 +1,362 @@
+"""
+That native debug better work!
+
+All these tests can be understood (somewhat) by running the equivalent
+`examples/debugging/` scripts manually.
+"""
+from os import path
+
+import pytest
+import pexpect
+
+from conftest import repodir
+
+
+# TODO:
+# - recurrent entry to breakpoint() from single actor *after* and an
```

Comment:
Actually, I think most of these are covered now but wouldn't hurt to do an audit in the future.

I don't have much juice left to keep pushing here :crying_cat_face: